packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -72,9 +73,15 @@ public void updateBuilder(CaptureRequest.Builder requestBuilder) {
     if (!checkIsSupported()) {
       return;
     }
-    requestBuilder.set(
-        CaptureRequest.CONTROL_AE_REGIONS,
-        exposureRectangle == null ? null : new MeteringRectangle[] {exposureRectangle});
+    if (defaultExposureRectangle == null) {
+      defaultExposureRectangle = requestBuilder.get(CaptureRequest.CONTROL_AE_REGIONS);
+    }
+    if (exposureRectangle != null) {
+      requestBuilder.set(
+          CaptureRequest.CONTROL_AE_REGIONS, new MeteringRectangle[] {exposureRectangle});
+    } else if (shouldReset(requestBuilder)) {
```

Comment:
In the case where `defaultExposureRectangle`, which seems to be the initial value of `requestBuilder.get(CaptureRequest.CONTROL_AE_REGIONS)`, is non-null, but `buildExposureRectangle()` then sets `exposureRectangle` back to `null` intentionally, will this work?

Reply:
Or is there no case where `exposureRectangle` should be set to null intentionally?

Reply:
Therefore, if defaultExposureRectangle is non-null and expositionRectangle is intentionally set to null, the code should still work correctly because shouldReset() will return false and the CONTROL_AE_REGIONS value will not be reset.

Reply:
> Or is there no case where `exposureRectangle` should be set to null intentionally?

If expositionRectangle is null, shouldReset() should correctly handle null or empty values.